.ci/travis/travis.sh

```diff
@@ -655,6 +655,39 @@ check-since-version)
   fi
   ;;
 
+cli-run-openjdk14)
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                 --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  CHECKSTYLE_DIR=$(pwd)
+  TRAVIS_DIR="${CHECKSTYLE_DIR}/.ci/travis"
+  NUMBER_OF_FILES=$(grep -c -v "#" "${TRAVIS_DIR}/openjdk14-test/jdk14-test-files.list")
```

Comment:
Used the `TRAVIS_DIR` variable here, since bash didn't like me splitting the lines of the subshell